### ISSUE TITLE
Latest release (8.5.r4.g4211bac7) includes garbage hidden files

### ISSUE THREAD

**Issue by rankynbass** [goatcorp/wine-xiv-git#9]
There's a bunch of hidden files in the current release (also in the unused 7.15 release). They're of the format `._file_or_folder_name` and contain some text of about Mac OS X and com.apple.quarantine. They don't actually seem to affect the functionality of wine, however.

```
Mac OS X        	2¢ÔATTRÔ<<com.apple.quarantineq/0083;6431d5c3;Safari;2A363FB3-DB59-4283-B02F-402D6BEDF207
```

The top level one is slightly different:

```
Mac OS X        	28ATTR8ŽŽHcom.apple.maclü<com.apple.quarantinekÔôÔML²ÃFÑÅ_q/0083;6431d5c3;Safari;2A363FB3-DB59-4283-B02F-402D6BEDF207
```

**Comment by NotNite** [comment]
This is information created by macOS - plenty of explanations online if you search "macOS dot underscore" (like [this one](https://github.com/ydkhatri/MacForensics/blob/master/DotUnderscore_macos.bt)). marzent is likely just a macOS user and messed up creating the archive. Whoops!

**Comment by marzent** [comment]
Yeah these ones are especially sneaky since they are representation of the file's resource fork/extended attributes if the native filesystem does not support them...
They are at least not garbage, do not affect wine at all and are transparently (de-)compressed with `bsdtar`:

```
% tar tf wine-xiv-staging-fsync-git-arch-8.5.r4.g4211bac7.tar.xz | grep ".*/\..*"
wine-xiv-staging-fsync-git-8.5.r4.g4211bac7/.INSTALL
wine-xiv-staging-fsync-git-8.5.r4.g4211bac7/.BUILDINFO
wine-xiv-staging-fsync-git-8.5.r4.g4211bac7/.MTREE
wine-xiv-staging-fsync-git-8.5.r4.g4211bac7/.PKGINFO
```

Anyways they still don't make too much sense here given the target os; I updated the release so they are no longer part of the archive (this interestingly saved up a few KB which is something I did not expect)
